storage/innobase/include/os0enc.h

```diff
@@ -83,6 +83,14 @@ class Encryption {
   information version. */
   static constexpr char KEY_MAGIC_V3[] = "lCC";
 
+#ifdef XTRABACKUP
+  static constexpr char KEY_MAGIC_PS_V3[] = "PSC";
+  /* To only  process crypt redo log generated by PS with encryption='n'*/
+  static constexpr uint KEYRING_INFO_MAX_SIZE = 102;
```

Comment:
I think this is 98 on PS better to use that. Maybe the entire length calculation as is